src/unix/linux_like/linux/gnu/mod.rs

```diff
@@ -542,6 +524,27 @@ impl siginfo_t {
     }
 }
 
+s_no_extra_traits! {
+    #[cfg_attr(feature = "extra_traits", derive(Debug))]
+    pub struct aiocb {
+        pub aio_fildes: ::c_int,
+        pub aio_lio_opcode: ::c_int,
+        pub aio_reqprio: ::c_int,
+        pub aio_buf: *mut ::c_void,
+        pub aio_nbytes: ::size_t,
+        pub aio_sigevent: ::sigevent,
+        __next_prio: *mut aiocb,
+        __abs_prio: ::c_int,
+        __policy: ::c_int,
+        __error_code: ::c_int,
+        __return_value: ::ssize_t,
+        pub aio_offset: off_t,
```

Comment:
Since this changes depending on file64, could you add this fixme so we catch it when doing that update?

```rust
        // FIXME(offset64): definition depends on oo64_t
        pub aio_offset: off_t,
```